src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Changes the remark of an existing person in the address book.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the list of persons "
+            + "by the alphabetical order of their name.\n"
+            + "Parameters: [-r]\n"
+            + "Example: " + COMMAND_WORD
+            + "-r";
+
+    private String message_success = "Sorted all persons";
+
+    private final boolean reverseOrder;
```

Comment:
I think changing this boolean variable name to isReverseOrder would improve code readability.